include/vinecopulib/bicop/implementation/fit_controls.ipp

```diff
@@ -96,9 +100,19 @@ FitControlsBicop::check_nonparametric_mult(double nonparametric_mult)
 inline void
 FitControlsBicop::check_selection_criterion(std::string selection_criterion)
 {
-    if (!tools_stl::is_member(selection_criterion, {"loglik", "aic", "bic"})) {
+    std::vector<std::string> allowed_crits = 
+        {"loglik", "aic", "bic", "mbic", "mbicv"};
+    if (!tools_stl::is_member(selection_criterion, allowed_crits)) {
         throw std::runtime_error(
-            "selection_criterion should be 'loglik', 'aic', or 'bic'");
+            "selection_criterion should be 'loglik', 'aic', 'bic', or 'mbic'");
+    }
+}
+
+inline void
+FitControlsBicop::check_psi0(double psi0)
+{
+    if (!(psi0 > 0.0) | !(psi0 < 1.0)) {
```

Comment:
psi0 = 1 would mean that we force non-indep, and psi0 = 0 the opposite, right? OK it may not be so useful.

Reply:
yeah we have `family_set` and `selection_criterion = "loglik"` for that.

Reply:
k